bin/yaml/julia.yaml

```diff
@@ -3,7 +3,8 @@ compilers:
     type: tarballs
     compression: gz
     dir: julia-{name}
-    check_exe: bin/julia --version
+    julia_exe: bin/julia
```

Comment:
Can I define custom fields in this yaml file?  I did it for convenience, to avoid repeating some things, but if it doesn't work I'll change this;

Reply:
you can; and that's the intention to let you cascading-ly refer to things later on as you do.